commcare_connect/opportunity/tw_tables.py

```diff
@@ -165,3 +171,644 @@ def render_end_date(self, value):
+               {% if value %}
+              {% include "tailwind/components/dropdowns/text_button_dropdown.html" with text=value.amount list=value.list styles='text-sm' %}
+          {% endif%}
+          </div>
+        """,
+    )
+
+    class Meta:
+        attrs = {"class": "w-full max-w-full", "thead": {"class": "hidden"}, "tbody": {"class": "block w-full h-full"}}
+        row_attrs = {
+            "class": "flex w-full justify-between gap-x-4 p-3 bg-white hover:bg-gray-100 relative transition-colors duration-300",
+        }
+        sequence = (
+            "index",
+            "opportunity",
+            "entityType",
+            "entityStatus",
+            "program",
+            "startDate",
+            "endDate",
+            "pendingInvites",
+            "inactiveWorkers",
+            "pendingApprovals",
+            "paymentsDue",
+        )
+
+    def render_index(self, value):
+        return format_html(
+            '<div class="flex justify-center text-sm font-normal truncate text-brand-deep-purple overflow-clip overflow-ellipsis w-9">{}</div>',
+            value,
+        )
+
+    def render_opportunity(self, value):
+        return format_html(
+            '<div class="flex justify-start w-40 text-sm font-normal truncate text-brand-deep-purple overflow-clip overflow-ellipsis">{}</div>',
+            value,
+        )
+
+    def render_program(self, value):
+        return format_html(
+            '<div class="flex justify-start text-sm font-normal truncate text-brand-deep-purple overflow-clip overflow-ellipsis w-36">{}</div>',
+            value,
+        )
+
+    def render_startDate(self, value):
+        return format_html(
+            '<div class="flex justify-center w-24 text-sm font-normal truncate text-brand-deep-purple overflow-clip overflow-ellipsis">{}</div>',
+            value,
+        )
+
+    def render_endDate(self, value):
+        return format_html(
+            '<div class="flex justify-center w-24 text-sm font-normal truncate text-brand-deep-purple overflow-clip overflow-ellipsis">{}</div>',
+            value,
+        )
+
+
+class WorkerPaymentsTable(tables.Table):
+    index = tables.Column(
+        orderable=False,
+        attrs={
+            "td": {
+                "class": "p-0",
+            }
+        },
+    )
+    worker = tables.Column(
+        verbose_name="Name",
+        attrs={
+            "td": {
+                "class": "p-0",
+            }
+        },
+    )
+    indicator = tables.TemplateColumn(
+        verbose_name="Indicator",
+        attrs={
+            "td": {
+                "class": "p-0",
+            }
+        },
+        orderable=False,
+        template_code="""
+                                    {% if value %}
+                                       <div class="w-[40px]"><div class="w-4 h-2 rounded bg-{{ value }}"></div></div>
```

Comment:
Since you are generating tailwind classes these dynamically, these are not getting picked up by tailwind. Please see documentation https://tailwindcss.com/docs/detecting-classes-in-source-files#dynamic-class-names

> Don't construct class names dynamically